cmd/svcat/output/binding.go

```diff
@@ -23,7 +23,7 @@ import (
 
 	"github.com/kubernetes-sigs/service-catalog/pkg/apis/servicecatalog/v1beta1"
 	svcatsdk "github.com/kubernetes-sigs/service-catalog/pkg/svcat/service-catalog"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Similar to `metav1`, let's consistently use `corev1` as an alias for consistency and clarity